misc/config_tools/xforms/lib.xsl

```diff
@@ -420,7 +420,7 @@
                 <node><xsl:value-of select="$res_clos_max"/></node>
               </xsl:when>
               <xsl:otherwise>
-                <node><xsl:value-of select="$res_clos_max div (1 + $cdp_enabled)"/></node>
+                <node><xsl:value-of select="floor(($res_clos_max + 1) div (1 + $cdp_enabled)) - 1"/></node>
```

Comment:
I don't think we need to invoke `floor` anymore. `$res_clos_max + 1` should always be a power of 2.